rotkehlchen/chain/ethereum/modules/lido_csm/balances.py

```diff
@@ -0,0 +1,116 @@
+from .metrics import LidoCsmMetricsFetcher
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.decoding.decoder import EthereumTransactionDecoder
+    from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+class LidoCsmBalances(ProtocolWithBalance):
+    """Queries Lido CSM accounting contract for node operator bond balances."""
+
+    def __init__(
+            self,
+            evm_inquirer: 'EthereumInquirer',
+            tx_decoder: 'EthereumTransactionDecoder',
+    ) -> None:
+        super().__init__(
+            evm_inquirer=evm_inquirer,
+            tx_decoder=tx_decoder,
+            counterparty=CPT_LIDO_CSM,
+            deposit_event_types=set(),
+        )
+        self._node_operator_db = DBLidoCsm(self.event_db.db)
+        self._steth_token = A_STETH.resolve_to_evm_token()
+        self._accounting_contract = EvmContract(
+            address=LIDO_CSM_ACCOUNTING_CONTRACT,
+            abi=ACCOUNTING_ABI,
+            deployed_block=0,
+        )
+        self._steth_contract = EvmContract(
+            address=self._steth_token.evm_address,
+            abi=STETH_ABI,
+            deployed_block=0,
+        )
+        # Reuse the metrics fetcher to compute pending rewards in stETH
+        self._metrics_fetcher = LidoCsmMetricsFetcher(evm_inquirer=evm_inquirer)
+        self._metrics_fetcher.steth_contract = self._steth_contract
+
+    def _get_bond_shares(self, node_operator_id: int) -> int:
+        return self._accounting_contract.call(
+            node_inquirer=self.evm_inquirer,
+            method_name='getBondShares',
+            arguments=[node_operator_id],
+        )
+
+    def _convert_shares_to_steth(self, shares: int) -> FVal:
+        return asset_normalized_value(
+            amount=self._steth_contract.call(
+                node_inquirer=self.evm_inquirer,
+                method_name='getPooledEthByShares',
+                arguments=[shares],
+            ),
+            asset=self._steth_token,
+        )
+
+    def query_balances(self) -> BalancesSheetType:
+        balances: BalancesSheetType = defaultdict(BalanceSheet)
+        node_operators = self._node_operator_db.get_node_operators()
+        if len(node_operators) == 0:
+            return balances
+
+        steth_price = Inquirer.find_usd_price(A_STETH)
+        for entry in node_operators:
+            # Fetch bond shares and convert to stETH
+            bond_steth = ZERO
+            try:
+                shares = self._get_bond_shares(entry.node_operator_id)
+                if shares != 0:
+                    bond_steth = self._convert_shares_to_steth(shares=shares)
+            except RemoteError as e:
+                log.error(
+                    'Failed to fetch/convert Lido CSM bond shares for node operator %s due to %s',
+                    entry.node_operator_id,
+                    e,
```

Comment:
```suggestion
                    f'Failed to fetch/convert Lido CSM bond shares for node operator {entry.node_operator_id} due to {e!s}',
```
Not a big deal, but we usually use f-strings instead of `%s`. I think you have this several places in this pr.